src/test/java/com/google/sps/TimingPropertiesTest.java

```diff
@@ -138,5 +138,55 @@ public void validParameters_for_userTookQuiz() {
         Boolean actual = timing_properties_test.userTookQuiz(user_quiz_time, game_quiz_time);
         Assert.assertEquals(false, actual);
     }
+
+    @Test
+    //Checks if the timestamp given is a null value
+    public void nullParameter_for_newDayNewQuiz(){
+        QuizTimingPropertiesUtils timing_properties_test = new QuizTimingPropertiesUtils();
+ 
+        Boolean actual = timing_properties_test.newDayNewQuiz(null);
+        Assert.assertEquals(null, actual);
+    }
  
+    @Test 
+    //Checks if newDayNewQuiz works especially with being dependent on the getTimestampProperty
+    public void validParameter_for_newDayNewQuiz() {
+        QuizTimingPropertiesUtils timing_properties_test = new QuizTimingPropertiesUtils();
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        Entity user = new Entity("user");
+ 
+        user.setProperty("quiz_timestamp", 159430944365L);
+        datastore.put(user);
+ 
+        Object user_quiz_time = timing_properties_test.getTimestampProperty("user", datastore);
+ 
+        Boolean actual = timing_properties_test.newDayNewQuiz(user_quiz_time);
+        Assert.assertEquals(true, actual);
+    }
+ 
+    @Test 
+    //If the Entity parameter is null then the getNewQuestion function should return null
```

Comment:
Comments shouldn't say the expected value, but instead just describe the inputs (this is a problem on one of my other PRs)
Ex. Test where the entity parameter is null

Reply:
Changed them! Let me know if the new ones are better